README.md

```diff
@@ -12,6 +12,9 @@ source for
   * [Antarctic (EPSG:3031)](https://nasa-gibs.github.io/gibs-web-examples/examples/openlayers/antarctic-epsg3031.html)
   * [Web Mercator (EPSG:3857)](https://nasa-gibs.github.io/gibs-web-examples/examples/openlayers/webmercator-epsg3857.html)
   * [Rolling Seven Day Slider](https://nasa-gibs.github.io/gibs-web-examples/examples/openlayers/time.html)
+  * [Vector Tile Basic Use](https://github.com/nasa-gibs/web-examples/blob/master/examples/openlayers/vectors/geographic-epsg4326-vector-basic.html)
```

Comment:
Should these also be linked from the `index.html` so you can easily get to them when running the examples locally?

Reply:
Yes, Definitely. Good find.